spec/spec.md

```diff
@@ -0,0 +1,180 @@
+<!--
+SPDX-Copyright: 2019 Free Software Foundation Europe e.V.
+
+SPDX-License-Identifier: CC-BY-SA-4.0
+-->
+
+# REUSE Specification
+
+This specification defines a standardized method for declaring copyright and
+licensing for software projects. The goal of the specification is to have
+unambiguous, machine-readable copyright and licensing information for each
+individual file in a project.
+
+## Definitions
+
+These are the definitions for some of the terms used in this specification:
+
+- Project --- ...
+
+- License file --- ...
+
+<!-- Also put a link to the SPDX Specification here somewhere -->
+
+<!-- Also DEP5 -->
```

Comment:
```suggestion
- DEP5 File --- `.reuse/dep5` file, which apart from its filename and location, follows the definition of [Machine-readable `debian/copyright` file, Version 1.0](https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/)
```

Reply:
We kind of do a subset of the DEP5 spec that possibly isn't entirely compatible. We mandate SPDX expressions in the REUSE spec, and those expressions are NOT in the DEP5 spec, but are supported by the library we use (`python-debian`).

We also mandate the format of the value following the Copyright tag, but the tool currently doesn't verify this.

Reply:
@carmenbianca, OK, that makes sense to me. But then it would make even more sense to also define the “REUSE DEP5” file, as the standard DEP5 with the following specifics, to make it more compatible with SPDX. If nothing else, to avoid confusion with the vanilla DEP5 files and spec.